File: site/code-snippets/Voice/getCallTranscription/sample.rb
require 'bandwidth'
  
include Bandwidth
include Bandwidth::Voice

bandwidth_client = Bandwidth::Client.new(
    voice_basic_auth_user_name: ENV['BW_USERNAME'],
    voice_basic_auth_password: ENV['BW_PASSWORD']
)

voice_client = bandwidth_client.voice_client.client

call_id = "c-1234"
recording_id = "r-1234"

begin
    response = voice_client.get_call_transcription(ENV['BW_ACCOUNT_ID'], call_id, recording_id)
    puts response.data.transcripts
rescue APIException => e
    puts e.response_code
end

File: site/code-snippets/Voice/getCall/sample.php
<?php

require "vendor/autoload.php";

$BW_USERNAME = getenv("BW_USERNAME");
$BW_PASSWORD = getenv("BW_PASSWORD");
$BW_ACCOUNT_ID = getenv("BW_ACCOUNT_ID");

$config = new BandwidthLib\Configuration(
    array(
        'voiceBasicAuthUserName' => $BW_USERNAME,
        'voiceBasicAuthPassword' => $BW_PASSWORD,
    )
);
$client = new BandwidthLib\BandwidthClient($config);

$voiceClient = $client->getVoice()->getClient();

$callId = "c-1234";

try {
    $response = $voiceClient->getCall($BW_ACCOUNT_ID, $callId);
    print_r($response->getResult()->state);
} catch (BandwidthLib\APIException $e) {
    print_r($e->getResponseCode());
}
